civis/cli/__main__.py

```diff
@@ -130,11 +136,12 @@ def param_case_map(param_names):
     return result
 
 
-def invoke(method, path, op, *args, json_output=False, **kwargs):
+def invoke(method, path, op, *args, **kwargs):
```

Comment:
Since `json_output` isn't explicit in the parameters any longer, you should mention it in a top-level comment or doc string.